### DISCUSSION TITLE
mysql-client in python fails on arm (Apple Silicon) mac

### DISCUSSION THREAD

**Discussion by johnsxiongs** [Homebrew/discussions#1426] [Everyday usage]
![image](https://user-images.githubusercontent.com/37213061/117542056-85debb80-b049-11eb-98fe-6fe13c91ce76.png)
![image](https://user-images.githubusercontent.com/37213061/117542246-6d22d580-b04a-11eb-846b-ed09f2dcc927.png)

Seems like it mismatched binaries. But I don't want  using an M1-native installation of Python.
https://github.com/Homebrew/discussions/discussions/569#discussioncomment-294050
How to solve this ?

**Comment by SMillerDev** [comment]
If you don't want native binaries, install homebrew using Rosetta.

**Reply by johnsxiongs** [reply]
> If you don't want native binaries, install homebrew using Rosetta.

Thanks for your answer! I just install x86_64 mysql version ,
![image](https://user-images.githubusercontent.com/37213061/117543772-cbeb4d80-b050-11eb-97a1-da73124ecc2b.png)

but it still doesn't work, what's the problem is !!

**Reply by SMillerDev** [reply]
What doesn't work, what are you trying to do? And please post text instead of screenshots.

**Reply by johnsxiongs** [reply]
> What doesn't work, what are you trying to do? And please post text instead of screenshots.

same as the first screenshots above, can't  import MySQLdb

```
Python 3.8.5 (default, Sep  4 2020, 02:22:02)
[Clang 10.0.0 ] :: Anaconda, Inc. on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import MySQLdb
Traceback (most recent call last):
  File "/opt/homebrew/anaconda3/lib/python3.8/site-packages/MySQLdb/__init__.py", line 18, in <module>
    from . import _mysql
ImportError: dlopen(/opt/homebrew/anaconda3/lib/python3.8/site-packages/MySQLdb/_mysql.cpython-38-darwin.so, 2): Symbol not found: _mysql_affected_rows
  Referenced from: /opt/homebrew/anaconda3/lib/python3.8/site-packages/MySQLdb/_mysql.cpython-38-darwin.so
  Expected in: flat namespace
 in /opt/homebrew/anaconda3/lib/python3.8/site-packages/MySQLdb/_mysql.cpython-38-darwin.so

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/homebrew/anaconda3/lib/python3.8/site-packages/MySQLdb/__init__.py", line 24, in <module>
    version_info, _mysql.version_info, _mysql.__file__
NameError: name '_mysql' is not defined
>>>
```

**Reply by SMillerDev** [reply]
This is much more likely an anaconda issue. You should try their support.
